model/job_queue/v1/job_resource.model

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages status of jobs on a job queue.
+resource Job {
+    // Mark a job as Successful
+    method Success {
+        in Body JobInstanceSuccess
```

Comment:
If you do this then the expected JSON request will be like this:

```json
{
  "body": {
    "receipt_id": "123"
  }
}
```

I assume that is not what you want, you probably want this:

```json
{
  "receipt_id": "123"
}
```

To achive that you have to define the method like this:

```
method Success {
        ReceiptId Integer
}
```

If you do that then you don't need the `JobInstanceSuccess` struct.

Same for the `Failure` method; you don't want/need the `JobInstanceFailure` struct.

Reply:
fixed